src/components/ExerciseThreeForm.js

```diff
@@ -7,23 +7,22 @@ const ExerciseThreeForm = () => (
     <div>
         <div>
             <h3>Exercise 3</h3>
-            <p>
-                I need to be a form that takes an input (using InputField component) name
-                and a textarea (using a TextareaField component that you have to create).<br />
-                Label for input is "Your name:" and label for textarea is
-                "Your text:"<br />
-                Please make sure the name is required and raise a warning if it is not only alphabetic string.<br />
-                The textarea field is required, must be alphanumeric and allow spaces, "'" and "." characters. Raise an error if one of these conditions isn't respected.<br />
-                You will raise a warning if the text doesn't contain the sentence 'This is super cool.'<br />
-                Result must show something like that:<br />
-            </p>
+            <p>Create a form that takes an input (using InputField component) with name="name", and a textarea (create your own TextareaField component) with name="text". </p>
```

Comment:
I've just added some formatting and modified the copy a bit, I just found it a little hard to read and follow the steps I was too take, so I just split it up so its a little easier to follow :)